test/jdk/sun/java2d/DirectX/OnScreenRenderingResizeTest/OnScreenRenderingResizeTest.java

```diff
@@ -74,126 +81,135 @@ public static void main(String[] args) {
                 System.err.println("Test will not check rendering results");
                 nocheck = true;
             } else {
-                System.err.println("Usage: OnScreenRenderingResizeTest [-inf][-nocheck]");
+                System.err.println("Usage: OnScreenRenderingResizeTest" +
+                                   " [-inf][-nocheck]");
             }
         }
 
-        BufferedImage output =
-            new BufferedImage(IMAGE_W, IMAGE_H, BufferedImage.TYPE_INT_RGB);
-        output.setAccelerationPriority(0.0f);
-        Graphics g = output.getGraphics();
-        g.setColor(renderColor);
-        g.fillRect(0, 0, output.getWidth(), output.getHeight());
-
-        final Frame frame = new Frame("OnScreenRenderingResizeTest") {
-            public void paint(Graphics g) {}
```

Comment:
Looks like these two empty methods are removed by the fix, why?

Reply:
We creating frame on EDT and running animation in main().
I didnt see any change by removing empty paint() method of frame(), thats why it is removed as cleanup.

Reply:
That empty methods were there to eliminate the clearing the frame by default implementation of it in Window/Container/Component.

Reply:
Okay. Since this is already integrated i have created https://bugs.openjdk.org/browse/JDK-8297241 to verify and update the test as per this thread.